packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -120,6 +67,11 @@ export class TimePicker extends LitElement {
 
   //#region Public Properties
 
+  /**
+   * @internal
+   */
+  @property() time: any = useTime(this);
```

Comment:
Couldn't figure out how to properly type this, so `any` is here to get past ts compliation errors.  Any help on this appreciated.

Reply:
Apologies, I just noticed the build errors, which are likely related to having the controller be a property. You can type it as:

```ts
@property() time: ReturnType<typeof useTime> = useTime(this);
```

Reply:
There's always something new to learn. ✨ I'd recommend TypeScript's own docs page: https://www.typescriptlang.org/docs/. The [handbook](https://www.typescriptlang.org/docs/handbook/intro.html) and reference sections are excellent and get updated often.